pip/commands/install.py

```diff
@@ -159,6 +159,14 @@ def __init__(self, *args, **kw):
 
         cmd_opts.add_option(cmdoptions.no_clean())
 
+        cmd_opts.add_option(
+            '--require-hashes',
```

Comment:
Should this be for ``pip install`` only? Does it make sense to be able to mandate hashes for ``pip download`` or ``pip wheel`` as well?